companyweb_base/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright 2021-2023 ACSONE SA/NV
```

Comment:
No copyright in trivial __init__.py files

Reply:
Done